Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] 
+    private int _speed = 3; 
+    Rigidbody2D rb2d;
+    // Start is called before the first frame update
+    void Start()
+    {
+        rb2d = GetComponent<Rigidbody2D>();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+       
+        if (Input.GetKey(KeyCode.A))
+        {
+            transform.position = transform.position + new Vector3(-(_speed)*Time.deltaTime,0,0);
+        }
+
+        if (Input.GetKey(KeyCode.D))
+        {
+           transform.position = transform.position + new Vector3(_speed*Time.deltaTime,0,0);
```

Comment:
Same as above and turn these two movement lines into a single function with parameter for direction. Can be `bool` or `int`.